src/services/ShSimulatorService.tsx

```diff
@@ -417,7 +417,7 @@ const ShSimulation = ({ children }: GenericContextProviderProps) => {
 			authKy
 				.delete(getEndpointFromSimulationInfo(info), {
 					signal,
-					searchParams: camelToSnakeCase(info.jobId)
+					searchParams: camelToSnakeCase({ jobId: info.jobId })
```

Comment:
I think this PR should be rebased, as this change has already been merged:
https://github.com/yaptide/ui/pull/1232/files